pulsar-websocket/src/main/java/com/yahoo/pulsar/websocket/WebSocketError.java

```diff
@@ -28,7 +28,9 @@
     FailedToDeserializeFromJSON(3, "Failed to de-serialize from JSON"), //
     FailedToSerializeToJSON(4, "Failed to serialize to JSON"), //
     AuthenticationError(5, "Failed to authenticate client"), //
-    NotAuthorizedError(6, "Client is not authorized"); //
+    NotAuthorizedError(6, "Client is not authorized"), //
+    InvalidPayload(7, "Invalid base64 payload"), //
```

Comment:
What about using PayloadEncodingError? 

My thinking is that the payload itself should never be invalid, since we don't care about it.